### ISSUE TITLE
Default Project Feature Request.

### ISSUE THREAD

**Issue by colton22** [Netflix-Skunkworks/go-jira#181]
Feature Request:
Ability to set default project when working with tickets. for example. I mainly work out of queue TEAM2. So all of my tickets are listed as:
```
TEAM2-1111
TEAM2-1523
TEAM2-1856
```
Would it be difficult to add a setting in ```~/.jira.d/config.yaml``` to specify ```DefaultProject: TEAM2```... 
my hopes is that I can run: ```gojira view 1111``` and view ticket TEAM2-1111... 

Logic behind this would be if ticket does not exist, if DefaultProject defined, try function again pre-pending DefaultProject.value. if ticket does not exist, error out.

This could save a lot of time. I can build a function around this but would like to see it a part of the program.

**Comment by coryb** [comment]
You can currently set `project: ...` in the `.jira.d/config.yml` file, and you can create that file per directory/project you work in.  I think this addresses the use case for your `DefaultProject` property.  

For the `jira view 1111` idea, I think we could get that to work. We can just run a simple regex against it to see if matches `^[0-9]+$` and automatically prepend the `project` property if set.  This should work with `jira 1111` as well.

**Comment by colton22** [comment]
Agreed; I didn't know the ```project: ...``` property existed already, that is easy to integrate than and yes, I feel the regex would work. I cannot seem to get this project to run so I can help in dev...

```
14:13:08 colton22:~/git/go-jira > go run cmd/jira/main.go 
# command-line-arguments
cmd/jira/main.go:50:28: cannot use fig (type *"github.com/coryb/figtree".FigTree) as type *"gopkg.in/Netflix-Skunkworks/go-jira.v1/vendor/github.com/coryb/figtree".FigTree in argument to jiracli.CommandLine
cmd/jira/main.go:50:28: cannot use o (type *"github.com/coryb/oreo".Client) as type *"gopkg.in/Netflix-Skunkworks/go-jira.v1/vendor/github.com/coryb/oreo".Client in argument to jiracli.CommandLine
14:13:23 colton22:~/git/go-jira > cat /etc/*release
Manjaro Linux
DISTRIB_ID=ManjaroLinux
DISTRIB_RELEASE=17.1.10
DISTRIB_CODENAME=Hakoila
DISTRIB_DESCRIPTION="Manjaro Linux"
Manjaro Linux
NAME="Manjaro Linux"
ID=manjaro
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="1;32"
HOME_URL="https://www.manjaro.org/"
SUPPORT_URL="https://www.manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
14:13:38 colton22:~/git/go-jira > 
```

**Comment by agzam** [comment]
Can someone please integrate this. It is a bit annoying to have to type something like `jira view BLABLA-1839`, I want to type `jira v 1839` instead.
